RecipeMatch/API/APIManager.m

```diff
@@ -96,7 +96,7 @@ - (void)getRecipesWithPreferences:(NSString * _Nullable)preferences andCompletio
     NSURL *url = [NSURL URLWithString:apiString];
     [self requestFromAPIWithURL:url andCompletion:^(NSDictionary *dataDictionary, NSError *error) {
         int count = [dataDictionary[@"count"] integerValue];
-        if (count > MIN_RECIPE_COUNT) {
+        if (count > MIN_RECIPE_COUNT) {  // if there are enough recipes, return them
```

Comment:
it might be worth adding more description here, like the logic and actual reasoning for it